decidim-core/decidim-core.gemspec

```diff
@@ -34,6 +34,7 @@ Gem::Specification.new do |s|
   s.add_dependency "high_voltage", "~> 3.0.0"
   s.add_dependency "date_validator", "~> 0.9.0"
   s.add_dependency "sprockets-es6", "~> 0.9.2"
+  s.add_dependency "cancancan", "~> 1.15.0"
```

Comment:
admin uses it, even if core has it, I still think each engine should declare its dependencies